osclib/comments.py

```diff
@@ -36,21 +57,21 @@ def _prepare_url(self, request_id=None, project_name=None,
             raise ValueError('Please, set request_id, project_name or / and package_name to add a comment.')
         return url
 
-    def _comment_as_dict(self, comment_element):
+    def _comment_as_dict(self, comment_element: ET.Element) -> Comment:
         """Convert an XML element comment into a dictionary.
         :param comment_element: XML element that store a comment.
         :returns: A Python dictionary object.
         """
-        comment = {
+        when_str = comment_element.get('when')
```

Comment:
Can this ever be None?

Reply:
No idea, there is zero documentation about the schemas of the comments API.

Reply:
In that case I recommend to remove the handling for `None` again. If the value is somehow missing it should fail fast and not at some point later when there's an unexpected missing timestamp somewhere...